### ISSUE TITLE
Playlist Editor - Import Icons

### ISSUE THREAD

**Issue by MikonosII** [DSheirer/sdrtrunk#819]
It would be a good option to have auto assignment of Icons.  Using the Radio Reference Function Tags are an easy way to assign icons.  Maybe have a user editable table that assigns Function Tags to Icons in the Icon Editor?

https://wiki.radioreference.com/index.php/Function_Tags

**Comment by jpryor223** [comment]
Agree.  Using the Function Tags for a default alias icon and color would be useful.

As a user, I want to manage default icons and colors based on Radio Reference Function Tags, so that I can import Radio Reference data with these defaults.

Possible similar use of Icon Manager for default icons and color based on Function Tags value:

![image](https://user-images.githubusercontent.com/66096304/83980479-dc90c200-a8db-11ea-8fd7-00caa899453b.png)

GIVEN the icon/function/color default manager WHEN viewing THEN default options are populated
GIVEN the icon/function/color default manager WHEN editing THEN function tag default icon and color options can be updated

As a user when importing from Radio Reference, I want to be to set talkgroup icons and color to defaults based on Function Tags, so that all talkgroups can be quickly imported with those settings.

Possible option:

Set Imported Talkgroups to Default Icons and Colors based on Function Tags

![image](https://user-images.githubusercontent.com/66096304/83980493-06e27f80-a8dc-11ea-8048-3e76f530c66a.png)

GIVEN the radio reference talkgroup view WHEN viewing THEN function tag values are visible in a column in the talkgroup list and field in the talkgroup details section
GIVEN the set imported talkgroups to defaults option is selected WHEN importing talkgroups from radio reference THEN the imported talkgroup aliases are set with the default icon and color

Possible icon and color defaults based on Radio Reference Function Tags value (tab delimited):

[Function Tags.txt](https://github.com/DSheirer/sdrtrunk/files/4743092/Function.Tags.txt)

**Comment by rd0fun** [comment]
+1
